samples/net/wifi/border_router/README.rst

```diff
@@ -0,0 +1,22 @@
+.. zephyr:code-sample:: openthread-border-router
+   :name: OpenThread Border Router
+   :relevant-api:
+
+   Test OpenThread Border Router functionality using the Wi-Fi shell module.
+
+Overview
+********
+TBD
+
+This sample allows testing OpenThrea Border Router. It also enables the net_shell module
+to verify net_if settings.
```

Comment:
The last sentence is probably not needed, net-shell is useful to have here for a sample that is basically just enabling options etc., so no need to specifically mention it.